Items/07_Items.txt

```diff
@@ -61,6 +61,33 @@ infection. For example, provides immunity from tear gas, The Doctor's
 Contaminate Area Procedure, etc. The suit provides no armor coverage and if the
 wearer takes damage the suit ruptures and no longer provides immunity.
 
+Opaleite Water Visor ($50):
+    A set of watertight goggles with a low-power filtration system and 
+rechargeable battery. When filled with water, an Opaleite wearing 
+the visor no longer suffers from racial Perception penalties from 
+being out of water. Provides no benefit to non-Opaleites.
+
+    Due to its bulk, wearing the visor precludes wearing any helmets or 
+other sizeable headgear.
```

Comment:
Precluding the use of other headgear (especially in this case Gas Masks) are a fantastic and intuitive penalty! Nice work!